scripts/dependency_ownership.js

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+require('../src/setup_node_env');
+
+require('@kbn/dependency-ownership/bin');
```

Comment:
`require('../src/setup_node_env')` should setup babel register already, is it still needed in this file?

Reply:
Addressed on e0ef65be9f8db2dbb929a61aa9291f9517a8fd02